package.json

```diff
@@ -26,16 +26,18 @@
     "express-rate-limit": "1.0.2",
     "extract-text-webpack-plugin": "0.8.0",
     "file-loader": "0.8.4",
+    "google-spreadsheet": "^1.0.1",
     "habitat": "0.4.1",
-    "hatchet": "0.3.2",
```

Comment:
yay problem solved! apologies if I didn't explain it clearly in the first place :stuck_out_tongue_winking_eye: